crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -1391,10 +1391,8 @@ fn handle_attribute_comment<'a>(
         .take_while(|token| token.kind == SimpleTokenKind::RParen)
         .last()
     {
-        return if comment.start() < right_paren.start() {
-            CommentPlacement::trailing(attribute.value.as_ref(), comment)
-        } else {
```

Comment:
We want to fall through to the next case if the comment is after the closing parentheses